### PR TITLE
[hotfix] [DOC] Fix missing areas in the development documentation #7959

### PR DESCRIPTION
**Pull request by YOMO-Lee**
Supplementary explanations have been provided for the methods and classes that must be implemented during the source development process

### CODE REVIEW

File: seatunnel-connectors-v2/README.zh.md [hunk 1]
```diff
@@ -39,14 +39,19 @@ SeaTunnel为与计算引擎进行解耦，设计了新的连接器API，通过
...
-
-​       package org.apache.seatunnel.connectors.seatunnel.{连接器名}.sink
+    package org.apache.seatunnel.connectors.seatunnel.{连接器名}.source
+    package org.apache.seatunnel.connectors.seatunnel.{连接器名}.sink
 
 4.将连接器信息添加到在项目根目录的plugin-mapping.properties文件中.
 
 5.将连接器添加到seatunnel-dist/pom.xml,这样连接器jar就可以在二进制包中找到.
 
+6.source端有几个必须实现的类，分别是{连接器名}Source、{连接器名}SourceFactor、{连接器名}SourceReader，具体可以参考其他连接器
```
Comment (davidzollo) [inline seatunnel-connectors-v2/README.zh.md]: missing character 'y',  {连接器名}SourceFactor ->  {连接器名}SourceFactory

File: seatunnel-connectors-v2/README.md [hunk 2]
```diff
@@ -134,7 +139,7 @@ completed by implementing this interface.
   these 100 pieces of data for batch processing. Stream processing does not have this requirement, so most SourceReaders
   with integrated stream batches will have the following code:
 
-```java
```
Comment (Edward-Gavin) [inline seatunnel-connectors-v2/README.md]: Why delete code type?

File: seatunnel-connectors-v2/README.zh.md [hunk 3]
```diff
@@ -103,7 +108,7 @@ SeaTunnel为与计算引擎进行解耦，设计了新的连接器API，通过
   中调用`SourceReader.Context.signalNoMoreElement`
   通知SeaTunnel没有数据读取了，那么就可以利用这100条数据进行批处理。流处理没有这个要求，那么大多数流批一体的SourceReader都会出现如下代码：
 
-```java
```
Comment (Edward-Gavin) [inline seatunnel-connectors-v2/README.zh.md]: Why delete code type?

**Comment by Hisoka-X** [comment]
duplicated of #8190

**Pull request closed without merging**